WORKSPACE

```diff
@@ -447,5 +466,53 @@ crates_repository(
 )
 
 load("@oak_crates_index//:defs.bzl", "crate_repositories")
+crate_repositories()
+
 
+load("@rules_rust//proto/prost:repositories.bzl", "rust_prost_dependencies")
+load("@rules_rust//proto/prost:transitive_repositories.bzl", "rust_prost_transitive_repositories")
+rust_prost_dependencies()
+rust_prost_transitive_repositories()
+crates_repository(
```

Comment:
I made a separate crates repository for toolchain management. I'm not sure if that's really needed, though.